### DISCUSSION TITLE
Add visual superset support

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#151] [Ideas]
Somehow add an ability to group exercises into a superset, so that'd be visible during a workout. Maybe something like ` / group: 1 /`, and all exercises with `group: 1` would be grouped into a superset.

**Comment by aryairani** [comment]
What do you think about doing something indented or with some other kind of marker like `{ }` or `---` so we don't have to make up a label or group number for each superset? I'm not sure how to deal with the fact that the number of sets should match across all:

```
Squat / 3x5
3x
  Bench Press / x8 / progress: dp(5lb, 8, 12)
  Bent Over Row, Barbell / x8
Crunch, Bodyweight / 3x20
```
```
Squat / 3x5
superset: 3x {
  Bench Press / x8 / progress: dp(5lb, 8, 12)
  Bent Over Row, Barbell / x8
}
Crunch, Bodyweight / 3x20
```
```
Squat / 3x5
---
Bench Press / 3x8 / progress: dp(5lb, 8, 12)
Bent Over Row, Barbell / 3x8
---
Crunch, Bodyweight / 3x20
```

**Comment by zedd3v** [comment]
Any updates on this ? @astashov

**Comment by ArthurianX** [comment]
@astashov  - I really like this app, today is the first time I saw time, I'm a long time engineer and developer, I'm willing to help if you want to / need a helping hand.

**Comment by RenanStigliani** [comment]
If this is implemented, "checking" a set as done, should jump to the next exercise in the superset

**Comment by mnadel** [comment]
I'd also love to see superset support!

**Comment by astashov** [comment]
This is done
